kakarot_scripts/ef_tests/fetch.py

```diff
@@ -32,20 +32,37 @@ def generate_tests():
         f"{os.path.basename(root)}_{name}": content
         for (root, _, files) in os.walk(EF_TESTS_DIR)
         for file in files
-        if file.endswith(".json") and "BlockchainTests/GeneralStateTests" in root
+        if file.endswith(".json")
+        and "BlockchainTests/GeneralStateTests" in root
+        and "Pyspecs" not in root
         for name, content in json.loads((Path(root) / file).read_text()).items()
         if content.get("network") == DEFAULT_NETWORK
     }
 
+    # Add tests from BlockchainTests/GeneralStateTests/Pyspecs
+    test_cases.update(
+        {
+            f"{name.split('::')[-1]}": content
+            for (root, _, files) in os.walk(EF_TESTS_DIR)
+            for file in files
+            if file.endswith(".json")
+            and "BlockchainTests/GeneralStateTests/Pyspecs" in root
+            for name, content in json.loads((Path(root) / file).read_text()).items()
+        }
```

Comment:
no need to filter on fork here?

Reply:
indeed, we can filter with `if f"fork_{DEFAULT_NETWORK}" in name` because pyspec tests have the fork in the name, like `src/test_access_list[fork_Cancun-blockchain_test]`